FunctionExecutor.gd

```diff
@@ -92,7 +92,13 @@ func tests_all_functions() -> void:
 						to_print += ")"
 						print(to_print)
 
-					object.callv(method_data["name"], arguments)
+					var ret = object.callv(method_data["name"], arguments)
+
+					if !BasicData.regression_test_project:
+						if ret != null && ret is Object:
+							if !(ret.get_class() in BasicData.disabled_classes) && \
+									!(ret.get_class() in ["PhysicsDirectSpaceStateSW", "Physics2DDirectSpaceStateSW"]):
+								BasicData.remove_thing(ret)
```

Comment:
Looks like deleting `PhysicsDirectSpaceState` cause crashes as well, I completely forgot to test with all Godot classes...

Note that `PhysicsDirectSpaceState` has `SW` appended, looks like these classes also cause problems according to the rest of the source code in Qarminer, so I've added exceptions here rather than adding those to `disabled_classes`.

Reply:
```
	var qq : BoxShape = null
	
	print(typeof(qq))
	print(TYPE_NIL)
``` 
both returns 0, so probably null check isn't necessary

Reply:
`ret is Object` will cause GDScript error if method that returns pointer to Object returns `nullptr`.